components/HotBillCard/Positions.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react"
+import { Container, CardImg } from "react-bootstrap"
+import Image from "react-bootstrap/Image"
+import styled from "styled-components"
+import ThumbsUp from "../../public/Thumbs Up.svg"
+
+export const Positions = (props: {
+  endorseCount: number
+  opposeCount: number
+  neutralCount: number
+}) => {
+  return (
+    <PositionsStyle>
+      <PositionStyle>
+        <p className="stanceTitle">Endorse</p>
+        <div>
+          <Image className="svg" alt="" src="Thumbs Up.svg" />
```

Comment:
1. include the leading slash , otherwise it's relative to the page it's displayed on
2. Let's rename these to thumbs-endorse.svg, thumbs-neutral.svg, and thumbs-oppose.svg for consistency with position

Reply:
I was not aware of that. If the component were pulled into another layout, the svgs would not be visible without the forward slash?